lib/hubspot/contact.rb

```diff
@@ -169,6 +170,15 @@ def is_new=(val)
       @is_new = val
     end
 
+    def emails
+      @identity_profiles
+        .pluck('identities')
+        .reject(&:blank?)
```

Comment:
`#blank?` is Rails-specific so we don't want it in this gem. Are you expecting `nil`s here or empty strings?